lib/miq_automation_engine/service_models/miq_ae_service_vm_or_template.rb

```diff
@@ -41,6 +41,7 @@ class MiqAeServiceVmOrTemplate < MiqAeServiceModelBase
     expose :compliances,           :association => true
     expose :last_compliance,       :association => true
     expose :ems_events,            :association => true
+    expose :evacuate,              :override_return => nil
```

Comment:
@andyvesel Why is the return being overridden here, can't the caller from the Automate Method just ignore it? This will cause us issues when we start implementing this in REST API.